.gitignore

```diff
@@ -3,3 +3,4 @@
 composer.lock
 cache
 *.sqlite
+.idea
```

Comment:
this is unnecessary, it should be in your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) file (check https://www.gitignore.io/ for OS/IDE related things)